From a93a0fd497c2e3deddbae4f5e76ed4b79b4a7e34 Mon Sep 17 00:00:00 2001
From: David Greaves <david.greaves@jolla.com>
Date: Wed, 27 May 2020 12:30:25 +0100
Subject: [PATCH 2/2] Reorganise vars and use LIBDIR for regulatory.bin

Signed-off-by: David Greaves <david.greaves@jolla.com>
---
 Makefile | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index a3ead30..ef595fb 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,3 @@
-# Modify as you see fit, note this is built into crda,
-# so if you change it here you will have to change crda.c
-REG_BIN?=/usr/lib/crda/regulatory.bin
-REG_GIT?=git://git.kernel.org/pub/scm/linux/kernel/git/sforshee/wireless-regdb.git
-
 PREFIX ?= /usr/
 MANDIR ?= $(PREFIX)/share/man/
 INCLUDE_DIR ?= $(PREFIX)/include/reglib/
@@ -10,6 +5,11 @@ LIBDIR ?= $(PREFIX)/lib
 
 SBINDIR ?= /sbin/
 
+# Modify as you see fit, note this is built into crda,
+# so if you change it here you will have to change crda.c
+REG_BIN?=$(LIBDIR)/crda/regulatory.bin
+REG_GIT?=git://git.kernel.org/pub/scm/linux/kernel/git/sforshee/wireless-regdb.git
+
 # Use a custom CRDA_UDEV_LEVEL when callling make install to
 # change your desired level for the udev regulatory.rules
 CRDA_UDEV_LEVEL?=85
-- 
2.20.1

